stardog-spring-batch/build.gradle

```diff
@@ -85,58 +85,62 @@ signing {
 	required { gradle.taskGraph.hasTask(uploadArchives) }
 }
 
-uploadArchives {
-	repositories {
-		mavenDeployer {
-			beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
-
-			repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
-				authentication(userName: sonatypeUsername, password: sonatypePassword)
-			}
-
-			pom.project {
-				name 'stardog-spring-batch'
-				packaging 'jar'
-				description 'Stardog Spring bindings for using Stardog with the Springframework Batch subproject'
-				url 'https://github.com/clarkparsia/stardog-spring/stardog-spring-batch'
-
-				scm {
-					url 'scm:git://github.com/clarkparsia/stardog-spring.git'
-					connection 'scm:git://github.com/clarkparsia/stardog-spring.git'
-					developerConnection 'scm:git://github.com/clarkparsia/stardog-spring.git'
-				}
-
-				licenses {
-					license {
-						name 'The Apache Software License, Version 2.0'
-						url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
-						distribution 'repo'
-					}
-				}
-
-				developers {
-					developer {
-						id 'albakercp'
-						name 'Al Baker'
-					}
+ uploadArchives {
+ 	repositories {
+ 		mavenDeployer {
+ 			beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+
+ 			repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+ 				authentication(userName: sonatypeUsername, password: sonatypePassword)
+ 			}
+
+ 			pom.project {
+ 				name 'stardog-spring-batch'
+ 				packaging 'jar'
+ 				description 'Stardog Spring bindings for using Stardog with the Springframework Batch subproject'
+ 				url 'https://github.com/clarkparsia/stardog-spring/stardog-spring-batch'
+
+ 				scm {
+ 					url 'scm:git://github.com/clarkparsia/stardog-spring.git'
+ 					connection 'scm:git://github.com/clarkparsia/stardog-spring.git'
+ 					developerConnection 'scm:git://github.com/clarkparsia/stardog-spring.git'
+ 				}
+
+ 				licenses {
+ 					license {
+ 						name 'The Apache Software License, Version 2.0'
+ 						url 'http://www.apache.org/licenses/LICENSE-2.0.txt'
+ 						distribution 'repo'
+ 					}
+ 				}
+
+ 				developers {
+ 					developer {
+ 						id 'albakercp'
+ 						name 'Al Baker'
+ 					}
+ 					developer {
+ 						id 'gcoluni'
+ 						name 'Gregory Coluni'
+ 					}
 					developer {
-						id 'gcoluni'
-						name 'Gregory Coluni'
+						id 'stardog_support'
```

Comment:
I would either put your name or something like stardog_oss, since "support" probably won't always maintain it

Reply:
I think that is the user Noah just created. Not sure if that would require all the jira ticket nonsense again.

Reply:
Yeah the user in Jira I created is `stardog_support`. I. think it has to be this unless I create a new account which isn't the end of the world.